eox_nelp/notifications/models.py

```diff
@@ -0,0 +1,45 @@
+"""
+Notifications models. This contains all models related with the notification.
+
+Models:
+    UpcomingCourseDueDate: Model that sends notifications based on the subsection stored data.
+"""
+import logging
+
+from django.db import models
+
+from eox_nelp.edxapp_wrapper.course_overviews import CourseOverview
+
+LOGGER = logging.getLogger(__name__)
+
+
+class UpcomingCourseDueDate(models.Model):
+    """Django model that stores necessary data to send email notifications to enrolled
+    users.
+
+    Fields:
+        course_id<Foreignkey>: Reference to a specific course.
+        location_id<CharField>: Store subsection id.
+        due_date<DateTimeField>: Subsection due date.
+        notification_date<DateTimeField>: Date when the notification must be sent.
+        sent<BooleanField>: Indicates if the notification was already sent.
+    """
+    course_id = models.ForeignKey(CourseOverview, null=False, on_delete=models.CASCADE)
+    location_id = models.CharField(max_length=255, null=False)
```

Comment:
Do you decide to avoid using the usage_key model field? I thinks because is becomes easier to develop and avoid crazy 500 management of usakge key shape?

Reply:
Indeed I thought the location that comes from the subsection was a simple string, but now I'm working with the receiver  and the location is an usage id therefore I'm going to update the model :(

Reply:
checking this also I think you can change course_id to course?
this due is strange access in this way.
![image](https://user-images.githubusercontent.com/51926076/232637905-5893a60d-3406-475d-9f80-3e30c153579b.png)

Reply:
course enrollments do it in that way.
![image](https://user-images.githubusercontent.com/51926076/232638097-fd83e5aa-b8ff-4028-b819-a8daeee70c60.png)